great_expectations/data_context/types/base.py

```diff
@@ -393,6 +399,8 @@ def validate_schema(self, data, **kwargs):
                 "ConfiguredAssetFilesystemDataConnector",
                 "InferredAssetS3DataConnector",
                 "ConfiguredAssetS3DataConnector",
+                "InferredAssetAzureDataConnector",
+                "ConfiguredAssetAzureDataConnector",
```

Comment:
Setting up the groundwork now so we don't have to make these changes in `InferredAssetAzureDataConnector`. Happy to remove those changes if it makes sense to put in the other connector's implementation.